packages/flutter_tools/lib/src/version.dart

```diff
@@ -118,21 +116,21 @@ class FlutterVersion {
         _channel = channel;
       }
     }
-    return _channel;
+    return _channel!;
```

Comment:
this means cast from `String?` -> `String`?

Reply:
Right, there's no code path where this is null.  Let me put it in a local to make it more obvious and enforced by the analyzer.

Reply:
Done.